install/kubernetes/helm/istio/values-istio-sds-auth.yaml

```diff
@@ -1,20 +1,30 @@
 global:
```

Comment:
this file is just for testing, right? can be removed.

Reply:
Yep, just for testing, will be removed before this graduates from draft